test/unit/storage/entities/account.js

```diff
@@ -34,16 +32,14 @@ describe('Account', () => {
 		it('should throw error for in-valid filters');
 		it('should accept only valid options');
 		it('should throw error for in-valid options');
-		it(
-			'should call adapter.executeFile with proper param for FIELD_SET_SIMPLE'
-		);
-		it('should call adapter.executeFile with proper param for FIELD_SET_FULL');
+		it('should call adapter.executeFile with proper param for extended=false');
+		it('should call adapter.executeFile with proper param for extended=true');
```

Comment:
This tests now need to be changed as now `get()` and `getOne()` both call `_getResults()` so I think the tests for `get()` and `getOne()` need to only check they're calling `_getResults()` with the valid parameters and then the heavy load on testing is for the implementation of that method. This change in tests should be implemented for all existing tests.

Reply:
As per talk this will be dealt with in different issue/PR